tutorials/dataframe/df103_NanoAODHiggsAnalysis.C

```diff
@@ -438,15 +438,15 @@ void df103_NanoAODHiggsAnalysis(const bool run_fast = true)
    const auto nevt_SMHiggsToZZTo4L = 299973.0; // H->4l: Number of simulated events
    const auto nbins = 36;                      // Number of bins for the invariant mass spectrum
    auto df_h_sig_4mu = df_sig_4mu_reco
-         .Define("weight", [&]() { return luminosity * xsec_SMHiggsToZZTo4L / nevt_SMHiggsToZZTo4L; }, {})
+         .Define("weight", [&]() { return luminosity * xsec_SMHiggsToZZTo4L / nevt_SMHiggsToZZTo4L; })
```

Comment:
since you are touching these lines we might as well also remove the other unnecessary brackes: `[&]()` -> `[&]`

Reply:
thnanks! Also, on few places `[]()` -> `[]` ... and also I missed some empty curly brackets. All fixed now